v3/plugins/redhat/quarkus-java8/1.5.0/meta.yaml

```diff
@@ -10,6 +10,9 @@ icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 repository: https://github.com/redhat-developer/vscode-quarkus
 category: Language
 firstPublicationDate: "2020-06-11"
+deprecate:
+  automigrate: true
+  migrateTo: redhat/quarkus-java11/latest
```

Comment:
does this also mean updating the quarkus devfile(s) so they're compatible with java11 instead of java8?

Reply:
Yep